### PR TITLE
Fix kafka topic config setter

### PR DESCRIPTION
**Pull request by neoeahit**
Refers internal ticket: KAFKA-5160

Currently this tool is broken, and this PR fixes this for both kafka 9 and kafka 10

### CODE REVIEW

File: kafka_utils/util/zookeeper.py [hunk 1]
```diff
@@ -134,9 +134,25 @@ def set_topic_config(self, topic, value):
...
+                    "entity_type": "topics",
+                    "entity_name": topic
+                })
+            elif version == 10:
+                # https://github.com/apache/kafka/blob/0.10.2.1/
+                #     core/src/main/scala/kafka/admin/AdminUtils.scala#L574
+                change_node = json.dumps({
+                    "version": 2,
+                    "entity_path": "topics/" + topic,
+                })
```
Comment (ecanzonieri) [inline kafka_utils/util/zookeeper.py]: This is missing an "else". What do we do if the version is neither 9 nor 10?

Comment (neoeahit) [inline kafka_utils/util/zookeeper.py]: added an assertion to tackle that case

File: kafka_utils/util/zookeeper.py [hunk 2]
```diff
@@ -134,9 +141,29 @@ def set_topic_config(self, topic, value):
                 config_data
             )
             # Create change
+            version = kafka_version[1]
+
+            # this feature is supported in kafka 9 and kafka 10
+            assert version in [9, 10], "Feature supported with kafka 9 and kafka 10"
```
Comment (ecanzonieri) [inline kafka_utils/util/zookeeper.py]: Let's use a set here: {9, 10}. It's doesn't make any difference in this case, but it's a bit nicer.

File: tests/util/zookeeper_test.py [hunk 3]
```diff
@@ -203,7 +203,7 @@ def test_set_topic_config(self, mock_client):
 
                 expected_create_call = mock.call(
                     '/config/changes/config_change_',
-                    "some_topic",
+                    json.dumps({"entity_path": "topics/some_topic", "version": 2}),
```
Comment (ecanzonieri) [inline tests/util/zookeeper_test.py]: I think we should have a unit test for 0.9 as well.

Comment (neoeahit) [inline tests/util/zookeeper_test.py]: added

File: kafka_utils/util/zookeeper.py (hunk 2 above)
Comment (neoeahit) [inline kafka_utils/util/zookeeper.py]: Discussed offline, and added ()

**Pull request merged**